cmd/entrypoint/secrets.go

```diff
@@ -313,6 +323,67 @@ func ReconcileSecrets(ctx context.Context, sh *SnapshotHolder) error {
 	return nil
 }
 
+// Returns secretName, secretNamespace from a provided (unstructured) filter if it contains a secret
+// Returns empty strings when the secret name and/or namespace could not be found
+func FindFilterSecret(filter *unstructured.Unstructured, action func(snapshotTypes.SecretRef)) error {
```

Comment:
exporting  for testing purposes?

I had a rant on my last review on this topic but keeping it short this time. In general, we should probably be cleaner about how what we export as our public API. However, I doubt there are people depending on emissary as a library...